tests/system/Database/Live/ForgeTest.php

```diff
@@ -678,6 +694,20 @@ public function testCompositeKey()
 			$this->assertEquals($keys['db_forge_test_1_code_active']->name, 'db_forge_test_1_code_active');
 			$this->assertEquals($keys['db_forge_test_1_code_active']->fields, ['code', 'active']);
 		}
+		elseif ($this->db->DBDriver === 'Sqlsrv')
```

Comment:
```suggestion
		elseif ($this->db->DBDriver === 'SQLSRV')
```

Reply:
Any reason to uppercase SQLSRV?

Reply:
The Driver name was mentioned in uppercase in php manual https://www.php.net/manual/en/book.sqlsrv.php also in Microsoft website reference https://docs.microsoft.com/en-us/sql/connect/php/sqlsrv-driver-api-reference?view=sql-server-ver15
so I think it would be nice to keep the driver name style in uppercase also its easier to read just like other drivers MySQL, SQLite3, PostgreSQL, etc..

Reply:
Personally, I don't think that this is worth changing. We already have other drivers that obviously don't follow the name convention from PHP. From what I know other frameworks also have their own names for database drivers. As long as there is no confusion about what driver it is, we probably should stay with the current name.